hack/ci/e2e.sh

```diff
@@ -187,9 +187,45 @@ EOF
         | grep -cv "node-role.kubernetes.io/master" \
     )"
 
+    KUBERNETES_PRESUBMIT_JOB=true
     # ginkgo regexes
-    SKIP="${SKIP:-}"
-    FOCUS="${FOCUS:-"\\[Conformance\\]"}"
+    if [[ "${KUBERNETES_PRESUBMIT_JOB:-}" = true ]]; then
+        # k/k presubmit jobs regex
+        SKIP="${SKIP:-"\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]"}"
```

Comment:
we need to collect this regex up and set it on the prowjob
as nice as it is to just checkout this repo and run `PRESUBMIT=true e2e.sh` or similar, we need what the skip is to be per-kubernetes branch going forward, as we may have to fix the _tests_ and the skip / focus will diverge.

this PR has been _super_ helpful to discover which tests though

Reply:
(and we should keep experimenting)

Reply:
Do you mean passing the SKIP var as an ENV variable in the prow job?